frictionless/project/database.py

```diff
@@ -281,3 +282,15 @@ def read_table(
         data = self.query(str(query))
         schema = record["resource"]["schema"]
         return ITable(tableSchema=schema, header=data["header"], rows=data["rows"])
+
+    def save_table(self, path: str, *, tablePatch: dict, basepath: str) -> Resource:
```

Comment:
Please add TODO that's it's temporary while we don't have proper database's table editing